sdk/schemaregistry/azure-schemaregistry-avroserializer/README.md

```diff
@@ -0,0 +1,122 @@
+# Azure Schema Registry Avro Serializer client library for Python
+
+Azure Schema Registry Avro Serializer provides the ability to serialize and deserialize data according
+to the given avro schema. It is integrated with Azure Schema Registry SDK and will automatically register and get schema.
+
+## Getting started
+
+### Install the package
+
+Install the Azure Schema Registry Avro Serializer client library for Python with [pip][pip]:
+
+```Bash
+pip install azure-schemaregistry-avroserializer
+```
+
+### Prerequisites: 
+To use this package, you must have:
+* Azure subscription - [Create a free account][azure_sub]
+* Azure Schema Registry
+* Python 2.7, 3.5 or later - [Install Python][python]
+
+### Authenticate the client
+Interaction with Schema Registry Avro Serializer starts with an instance of SchemaRegistryAvroSerializer class. You need the endpoint, AAD credential and schema group name to instantiate the client object. 
+
+**Create client using the azure-identity library:**
```

Comment:
Got feedback from cala; we're not supposed to have samples in these sections, we could make it a formal sample and link to it however, which is what I've leaned towards.  Feel free to eyeball my [current SB PR](https://github.com/Azure/azure-sdk-for-python/pull/13511) for an example of this.